app/components/Dashboard/PriceHistoryPanel/PriceHistoryChart.jsx

```diff
@@ -103,7 +105,7 @@ export default class PriceHistoryChart extends React.Component<Props, State> {
   }
 
   tooltipContent = (tooltipProps: Object) =>
-    !!tooltipProps.payload.length && (
+    !!get(tooltipProps, '.payload.length') && (
```

Comment:
I think it should be `get(tooltipProps, 'payload.length')` <- without the leading '.' before `payload`